fem/fespace.hpp

```diff
@@ -169,31 +204,81 @@ class FiniteElementSpace
        boundary. */
    void BuildNURBSFaceToDofTable() const;
 
+   /// Bit-mask representing a set of orders needed by an edge/face.
+   typedef std::uint64_t VarOrderBits;
+   static constexpr int MaxVarOrder = 8*sizeof(VarOrderBits) - 1;
+
```

Comment:
Is `MaxVarOrder = 63` adequate for all use cases. If it is not large enough, will there be an informative error or a seg-fault?

Reply:
Hopefully this is more than enough for most uses. There is an `MFEM_VERIFY` in `SetElementOrder` so there shouldn't be a mysterious failure if the user tries to set an order that is too high. If this ever is not enough, we can come up with a solution. Right now I think this is sufficient *and* many times more efficient than e.g. having an `std::set` for each edge and face.